client/src/components/Questions/Questions.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import QuestionWithAnswer from './QuestionWithAnswer';
+import QuestionWithoutAnswer from './QuestionWithoutAnswer';
+
+function Questions({ details }) {
+  const questions = details.map((question) => {
+    if (question.answer) {
+      return (
+        <div>
```

Comment:
nit: Not sure if `<div>` wrapper is required since you are returning a rendered component

Reply:
You're right, wasn't— took that out.